go/materialize/driver/image/driver.go

```diff
@@ -39,7 +39,7 @@ type driver struct {
 }
 
 // NewDriver returns a new Docker image driver.
-func NewDriver(networkName string) pm.DriverServer { return driver{} }
+func NewDriver(networkName string) pm.DriverServer { return driver{networkName: networkName} }
```

Comment:
whoops thanks. Linter caught ?

Reply:
I got that one in the process of testing the BigQuery adapter.